sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/BuilderHelper.java

```diff
@@ -99,6 +100,16 @@ static HttpPipeline buildPipeline(TablesSharedKeyCredential tablesSharedKeyCrede
             .build();
     }
 
+    static HttpPipeline buildNullClientPipeline() {
+        List<HttpPipelinePolicy> policies = new ArrayList<>();
```

Comment:
Since there is only one element, you probably don't have to allocate a list for it:

```java
HttpPipelinePolicy[] policies = {
    new AddHeadersPolicy(new HttpHeaders().put("Accept", "application/json;odata=minimalmetadata"))
};
```